Basic-Car-Maintenance/Documentation.docc/getting-started.tutorial

```diff
@@ -108,4 +108,44 @@
             }
         }
     }
+
+    @Section(title: "Setting Up Firebase Local Emulator") {
+        @ContentAndMedia {
+            Follow these steps to set up the Firebase Local Emulator to load data locally and not affect production. Please do not skip this step.
+        }
+
+        @Steps {
+            @Step {
+                Install Homebrew, a package manager for macOS, if you haven't already:
+                `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
+            }
+            @Step {
+                Install Xcode command line tools:
+                `xcode-select --install`
+            }
+            @Step {
+                Install Node Version Manager (NVM):
+                `brew install nvm`
+                Installs node version manager, so you don't need to update the system node version.
+            }
+            @Step {
+                Add the executable to the `$PATH` via `.zshrc` or `.bashrc` file as prompted after installation. Do NOT forget this! (and then restart your Terminal)
+            }
+            @Step {
+                Download and use the latest stable version of Node.js:
+                `nvm install stable`
+                `nvm use stable`
+            }
```

Comment:
same here, also notice how in the docs you can't tell that these are two commands, and you should (see screenshot), it looks like one command, because the line break doesn't show)

![Screenshot 2024-10-01 at 7 15 29 PM](https://github.com/user-attachments/assets/ead37b3e-b9ac-4698-8127-cb4ce30a2517)